unix/x0vncserver/x0vncserver.man

```diff
@@ -125,8 +125,9 @@ parameter instead.
 .B \-PlainUsers \fIuser-list\fP
 A comma separated list of user names that are allowed to authenticate via
 any of the "Plain" security types (Plain, TLSPlain, etc.). Specify \fB*\fP
-to allow any user to authenticate using this security type. Default is to
-deny all users.
+to allow any user to authenticate using this security type. Specify \fB%u\fP
+to allow the effective user of the server process. Default is to deny all
+users.
```

Comment:
Great that you also remembered the documentation!

Reply:
I think we need to remove "effective" from the description, since we are using `getuid()`, not `geteuid()`. I will submit a follow-up PR.